src/main.cc

```diff
@@ -184,9 +184,16 @@ main(const int argc, char* argv[]) {
         .is_err();
   } catch (const structopt::exception& e) {
     if (argc > 1) {
+      int subcommand_index = 1;
+      for (; subcommand_index < argc; ++subcommand_index)
+        if (argv[subcommand_index][0] != '-')
+          break;
```

Comment:
```suggestion
      for (; subcommand_index < argc; ++subcommand_index) {
        if (argv[subcommand_index][0] != '-') {
          break;
        }
      }
```

While being something redundant, surrounding with `{}` would be more explicit.